server/api/db/sqldb/models.py

```diff
@@ -769,6 +771,61 @@ class TimeWindowTracker(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.key}"
 
+    class ModelEndpoint(Base, mlrun.utils.db.HasStruct):
+        __tablename__ = "model_endpoints"
+        __table_args__ = (
+            UniqueConstraint(
+                "project", "name", "uid", name="_mep_uc_2"
+            ),  # create time? in order to save old MEP
+            ForeignKeyConstraint(
+                ("function_name", "function_uid", "project"),
+                ["functions.name", "functions.uid", "functions.project"],
+                name="_mep_function_constraint",
+            ),
+            ForeignKeyConstraint(
+                ("model_uid",), ["artifacts_v2.uid"], name="_mep_model_constraint"
+            ),
```

Comment:
Should we also consider adding a constraint with the model monitoring feature set? I’m not sure if it’s possible since we just store a reference to the feature set URI.